File: SDKSampleJava/app/src/main/java/com/gyant/sdksamplejava/DisplayGyantChatActivity.java
package com.gyant.sdksamplejava;

import android.os.Bundle;

import com.gyant.gyantchatsdk.GyantChat;
import com.gyant.gyantchatsdk.GyantChatActivity;

public class DisplayGyantChatActivity extends GyantChatActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        //gyantChatInit should be called before onCreate
        GyantChat.getInstance()
                .clientId("client_id")
                .patientId("patient_id").start();

        super.onCreate(savedInstanceState);
    }
}

File: SDKSampleKotlin/app/src/main/java/com/gyant/sdksamplekotlin/DisplayGyantChatActivity.kt
package com.gyant.sdksamplekotlin

import android.os.Bundle

import com.gyant.gyantchatsdk.GyantChat
import com.gyant.gyantchatsdk.GyantChatActivity


class DisplayGyantChatActivity : GyantChatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        val gyantChat = GyantChat.getInstance()
            .clientId("client_id")
            .patientId("patient_id")
            .isDev(true)
            .start()
        super.onCreate(savedInstanceState)
    }
}
